modules/ROOT/pages/markdown.adoc

```diff
@@ -0,0 +1,91 @@
+= Markdown plugin
+:navtitle: Markdown
+:description: The Markdown Premium plugin detects pure markdown from a paste event within a TinyMCE editor instance.
+:description_short: Convert pure markdown on paste.
+:keywords: plugin, Markdown.
+:pluginname: Markdown
+:plugincode: markdown
+:plugincategory: premium
+
+include::partial$misc/admon-paid-addon-pricing.adoc[]
+
+include::partial$misc/admon-requires-7.0v.adoc[]
+
+The {pluginname} Premium plugin detects pure markdown from a paste event within a {productname} editor instance.
+
+== Using the {pluginname} plugin
+
+
+=== Setup
+
+// TODO
+
+=== How it works
+
+The **Markdown** Premium Plugin allows the editor to detect if pure markdown is pasted into the {productname} editor instance.
+
+During this paste event, If the editor detects markdown has been pasted it will:
+
+* attempt to convert the markdown to HTML, while
+* add a undo level before the conversion to allow for undo of plain-text.
+
+=== What {pluginname} handles
+
+//TODO
```

Comment:
Should we cut off this section?

Reply:
Yes, this was just from the initial template.

Reply:
```suggestion
```